### DISCUSSION TITLE
Soliciting Topics for Knative Project Meeting @ KubeCon NA 2022

### DISCUSSION THREAD

**Discussion by lance** [knative/community#1172] [Q&A]
# Project Meeting at KubeCon

We will be having a Knative Project meeting at KubeCon NA in Detroit on Tuesday, October 25 from 1-5pm. We have a room with enough space for 30 people. This is an opportunity to meet and work face to face with many of the people you do the Slack dance with daily!

The conference organizers will be providing power, beverages and a white board. This is meant to be a working meeting among collaborators and people who would like to become collaborators.

## Will you be there?

Reply in comments if so! Known attendees so far:

* @evankanderson Evan Anderson
* @psschwei Paul Schweigert
* @lionelvillard Lionel Villard
* @rhuss Roland Huß
* @lance Lance Ball
* @nainaz Naina Singh
* @maximilien Dr. Max
* @csantanapr Carlos Santana
* @dsimansk David Simansky
* @aslom Aleksander Slominski

## Proposed Topics (so far):

If you have something you would like to add to the agenda, now is your chance. Reply in the comments.

* Eventing security
* Discuss better integration with the larger community on things like `pkg`
* Consider CI/CD as a delivery target
* Community metrics / OTel
* After Functions GA - what next - build strategy, eventing, tooling, etc.
* Knative as the API for Cloud FaaS (AWS Lambda, Google Cloud Functions, etc)
* Knative Catalog (plugins, sources)
* Future of `kn` + plugins (feature completion & maintenance)
* Knative Everywhere
* Knative Event Sources

/cc @knative/steering-committee  @knative/technical-oversight-committee @knative/knative-admin

**Comment by nainaz** [comment]
Please add me @nainaz 
Apart from these topics. 
Performance evaluation

**Comment by lionelvillard** [comment]
I'd like to talk about source conformance.

**Comment by maximilien** [comment]
+1 on performance and specifically our old friend “cold-start” performance

**Comment by csantanapr** [comment]
Please add me, I will be there

topic: Knative as the API for Cloud FaaS (AWS Lambda, Google Cloud Functions, etc)

**Comment by dsimansk** [comment]
I'll be there, please add me.

Topics: Knative Catalog (plugins, sources?), future of `kn` + plugins (feature completion & maintenance).

**Comment by aslom** [comment]
Please add me as well

Topics: Knative everywhere, in particular running Knative Eventing across multiple big and very small clusters
